apps/demos/src/zone/zone-flags-configurator.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ZoneGlobalDisableConfigurationsKey,
+  zoneGlobalDisableConfigurationsKeys,
+  ZoneRuntimeConfigurationsKey,
+  zoneRuntimeConfigurationsKeys,
+  ZoneTestDisableConfigurationsKey,
+  zoneTestDisableConfigurationsKeys,
+  ZoneTestSettingsConfigurationsKey,
+  zoneTestSettingsConfigurationsKeys
+} from './model/configurations.types';
+import { ZoneGlobalConfigurations } from './model/zone.configurations.api';
+
+type GlobalDisableConfigurationMethods = {
+  [disabledFlag in ZoneGlobalDisableConfigurationsKey]: (isDisabled?: boolean) => void;
+};
+
+type TestDisableConfigurationMethods = {
+  [disabledFlag in ZoneTestDisableConfigurationsKey]: (isDisabled?: boolean) => void
+} & {
+  [symbolFlag in ZoneTestSettingsConfigurationsKey]: (isDisabled?: boolean) => void
+};
+
+type targetSet = [WebSocket, (keyof WebSocketEventMap)[]] |
+  [any, (keyof WindowEventMap)[]];
+type GlobalTargetConfigurationMethod = (...args: targetSet) => void;
+type RuntimeConfigurationMethods = {
+  [disabledFlag in ZoneRuntimeConfigurationsKey]: (isDisabled?: boolean) => void;
+}
+
+const zoneDisable = '__Zone_disable_';
+const zoneIgnore = '__Zone_ignore';
+const zoneSymbol = '__zone_symbol__';
+
+
+export interface ZoneFlagsConfigurator {
+  global: {
+    disable: GlobalDisableConfigurationMethods
+  },
+  test: {
+    disable: TestDisableConfigurationMethods
+  },
+  target: {
+    disable: GlobalTargetConfigurationMethod
+  }
+  runtime: {
+    disable: RuntimeConfigurationMethods
+  },
+}
+
+function getZoneFlagsConfigurator(): ZoneFlagsConfigurator {
```

Comment:
```suggestion
function createZoneFlagsConfigurator(): ZoneFlagsConfigurator {
```

I think this is a creator function. So let's name it like one :)